department-of-veterans-affairs/va.gov-cms
# VAGOV-4034

- Changes service offering field method to unpublished default.

Task: https://va-gov.atlassian.net/browse/VAGOV-4034

**QA:** 

- Visit `admin/structure/types/manage/health_care_local_facility/fields/node.health_care_local_facility.field_local_health_care_service_`
- Visually verify that field method is set to `Unpublished default`
